rna-transcription/example.nim

```diff
@@ -0,0 +1,19 @@
+proc toRna*(strand: string): string =
+  var rna = ""
+
+  for c in strand:
+    if c == 'A':
+      rna &= 'U'
+    elif c == 'G':
+      rna &= 'C'
+    elif c == 'C':
+      rna &= 'G'
+    elif c == 'T':
+      rna &= 'A'
+    else:
+      var err: ref ValueError
```

Comment:
this can be shortened to: ``raise newException(ValueError, "Invalid nucleotide")``.